### PR TITLE
Minor clarifications for P88 and P145

### PR DESCRIPTION
**Pull request by prabau**
Minor clarifications for P88 and P145.

Also trivial tweaks for two theorems.

### CODE REVIEW

File: properties/P000088.md [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
     name: Counterexamples in Topology
   - wikipedia: Collectionwise_normal_space
     name: Collectionwise Normal Space on Wikipedia
   - zb: "0684.54001"
     name: General Topology (Engelking, 1989)
+  - zb: "0529.54017"
+    name: Collectionwise Hausdorff versus collectionwise normal with respect to compact sets (Reed)
 ---
 
 For every discrete family $(F_i)_{i \in I}$  of closed subsets of $X$ there exists a pairwise disjoint family of open sets $(U_i)_{i \in I}$, such that $F_i \subseteq U_i$.
```
Comment (GeoffreySangston) [inline properties/P000088.md:17]: Would the following read better?

For every discrete family $(F_i)_{i \in I}$  of closed subsets of $X$, there exists a pairwise disjoint family of open sets $(U_i)_{i \in I}$ such that $F_i \subseteq U_i$ for all $i$.

Comment (prabau) [reply properties/P000088.md:17]: Definitely.  Making the change.

**Review by GeoffreySangston — APPROVED** [review]
I saw no mistakes.

**Comment by GeoffreySangston** [comment]
I just saw that in some sources, the $(U_i)$ from the primary definition of CWN is called a *disjoint open expansion* of $(F_i)$, and the $(U_i)$ from the secondary definition is called a *discrete open expansion* of $(F_i)$. These seem relatively common (["discrete open expansion"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C21&q=%22discrete+open+expansion%22&btnG=), ["disjoint open expansion"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C21&q=%22disjoint+open+expansion%22&btnG=)), so I guess they are standard. There's also locally-finite open expansion.

I guess you may not have included these since the first line already defines discrete family. Would these be clarified at all by giving the first definition as "Every discrete family of closed sets has a discrete open expansion. A *discrete family* is... A discrete open expansion is...". Then give secondary definition as "A discrete family has a disjoint open expansion. A *disjoint open expansion* is..."? Or maybe open expansion should be defined, and then discrete / disjoint open expansion? It might be too much.

**Comment by prabau** [comment]
Good to know about these terms.  The few articles I checked in the links you provided seem to all define what they mean by these "expansions", so it may not be a universally understood terminology.  I am not so sure there is a need to add them here though, as it if not an alias for the main property being defined here.  What we need here is some clear description of the properties we are defining, which we do have.  Maybe we can leave it as is for now?

**Comment by prabau** [comment]
@GeoffreySangston FYI, I am adding two theorems having to do with CWN, and using "discrete open expansion" and "disjoint open expansion" seems very convenient when doing some proofs.  So I'm going to add that at the same time.

**Comment by prabau** [comment]
Actually, I can't find a good reference for "disjoint open expansion" in this context.  Any suggestion?

**Comment by GeoffreySangston** [comment]
@prabau 

The following are not ideal I think, but are the best I'm finding. 'Expansion' seems to be older than all of these. These define an 'expansion' of a collection as a kind of collection, and apply 'open', 'pairwise disjoint' = 'disjoint', and 'discrete' as modifiers to a collection. 

- [Gruenhage-Nogura-Purisch Normality of](https://www.sciencedirect.com/science/article/pii/0166864191901197) $X \times \omega_1^\ast$. See just before Lemma 2.1. This one explicitly defines 'open' and 'disjoint' for a collection just before the definition of expansion. I don't think the paper defines collectionwise normal.
- [Price's dissertation](https://vtechworks.lib.vt.edu/items/1061a224-0465-40e1-b00b-14df0a52c28c) defines 'collectionwise normal (CWN)' in Definition 1.2.36. on page 20 using the 'disjoint open expansion' concept, and defines expansion in Definition 1.2.34. 
- [Fast's dissertation](https://www.proquest.com/docview/303884140?pq-origsite=gscholar&fromopenview=true&sourcetype=Dissertations%20&%20Theses) defines expansion in Definition 1.2.27 and defines CWN in Definition 1.2.30 using 'pairwise disjoint open expansion'.

**Comment by prabau** [comment]
The Gruenhage-Nogura-Purish article actually has a different meaning of "disjoint", for two collections of sets indexed by a common index set.  And I am not sure we want to refer to theses to indicated notability.

In the literature, it seems that "disjoint open expansion" is mostly used for the case where each closed set in the family is a singleton.  That's related to the notion of "collectionwise Hausdorff" and variants.
For example in https://zbmath.org/1168.54008 and https://zbmath.org/1410.54025.

**Comment by GeoffreySangston** [comment]
@prabau 

> Good to know about these terms. The few articles I checked in the links you provided seem to all define what they mean by these "expansions", so it may not be a universally understood terminology.

From Junnila's [On Submetacompactness](https://topology.nipissingu.ca/tp/reprints/v03/tp03207.pdf), which is one of the references at [Submetacompact P000194](https://topology.pi-base.org/properties/P000194):
>  In [33], Worrell and Wicke extended earlier results of Michael and Nagami ([22] and [26]) and McAuley ([21]) with the following theorem: every collectionwise normal submetacompact Hausdorff-space is paracompact.

That appears to be missing from pi-base: [π-Base, Search for `submetacompact + hausdorff + Collectionwise normal + ~Paracompact`](https://topology.pi-base.org/spaces?q=submetacompact+%2B+hausdorff+%2B+Collectionwise+normal+%2B+%7EParacompact)

Should that be added? Also, I noticed Junnila's paper defines 'expansion' and uses 'open expansion'.

(I just saw this because I was wondering in what context 'submetacompact' is studied while following a discussion you and David were having.)

**Comment by prabau** [comment]
[ Collectionwise normal + submetacompact + T2 => paracompact ]
It's not an urgent thing, but I think it would be nice to have at some point.  Among other things, it would allow to derive that a few spaces, like the ordinal space $\omega_1$, are not submetacompact.

That theorem seems to be Theorem (iii) on p. 825 of the Wicke-Worrell paper.  The problem is that they don't give any detail.  They just claim it can be proved by methods similar to other results in other papers.  As it does not seem an urgent thing, maybe we can leave this alone for now, until someone wants to investigate and pursue this to completion.

As for why submetacompactness is important, I am no expert.  @ccaruvana introduced the shrinking property and added submetacompact because it was related to the shrinking property.  That's when I learned about the concept.  It falls into an array of related covering properties, as explained for example in Handbook of set-theoretic topology. (Long term, these would be good to have in pi-base; but I would not recommend adding all of these right now, unless someone has a need for them.)

As an aside, I looked at the reference for https://topology.pi-base.org/theorems/T000545 [submetacompact + normal => shrinking] and tried to make sense of it.  But it was nothing obvious, as it relied on lemmas in inaccessible sources.  Later on, I happened upon Yasui's thesis, which was able to fill part of the gap.  I should go back and add that as an additional reference at some point.

**Pull request merged**